src/OrchardCore.Modules/OrchardCore.Workflows/Startup.cs

```diff
@@ -29,6 +30,10 @@ namespace OrchardCore.Workflows;
 
 public sealed class Startup : StartupBase
 {
+    private readonly IShellConfiguration _shellConfiguration;
+
+    public Startup(IShellConfiguration shellConfiguration) => _shellConfiguration = shellConfiguration;
```

Comment:
OK, but isn't this supposed to show up as a violation somewhere? Because not even in the editor window has it (the build won't due to the IDE* issue).

Reply:
I'm not sure why we don't use body expression whenever it's possible!!

Reply:
body expressions are helpful when the code is short. But in a constructor, I don't think they make sense. Also if someone wants to inject new service, the have to convert it back. Personally I like to see the constructor and easily identify that it is the constructor. I know at least @sebastienros and @gvkries agree with my here since we just have this discussion couple hours ago.

Reply:
> But in a constructor, I don't think they make sense.

Why?

> Personally I like to see the constructor and easily identify that it is the constructor.

It's your personal thought :) what if you are using primary constructor?

Reply:
Please take this conversation to the discussion board or the issue tracker, since it goes well beyond this PR. I'm reverting this to a standard ctor.

Reply:
> what if you are using primary constructor

just don't :)  Also we have a rule in the `.editorconfig` against them